src/main/java/session/SessionDb.java

```diff
@@ -0,0 +1,18 @@
+package session;
+
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+
+import session.Session;
+
+public class SessionDb {
+    private static Map<String, Session> sessionMap = Maps.newHashMap();
+
+    public static String addSessionedUser(String userId) {
```

Comment:
```suggestion
    public static String addSessionedUser(String sessonId) {
```
정확하게는 UserId 가 아니고 sessionId 겠죠?